### ISSUE TITLE
Do not include js in the repository

### ISSUE THREAD

**Issue by max-nextcloud** [nextcloud/app-tutorial#476]
This tutorial should be used to develop apps that will most likely not be shipped with the server. So there's no need to include the compiled js.

We should avoid including the js in the repo when possible because:
* It leads to conflicts on rebases that would otherwise be smooth.
* It requires rebuilding the js as a production build for every pull request.
* It requires additional consistency checks in ci to ensure js is updated.
* It increases the repo size significantly.

**Comment by susnux** [comment]
I think ´js/` is included so users can directly test it without installing node dependencies and building the js assets:
https://github.com/nextcloud/app-tutorial#try-it

**Comment by max-nextcloud** [comment]
> I think ´js/` is included so users can directly test it without installing node dependencies and building the js assets: https://github.com/nextcloud/app-tutorial#try-it

Good point. On the other hand if you want to make changes you will need to compile the js anyway. And if people use this as a template for their own app they will start with an included js which can be annoying.

So I agree... not an obvious decision here.
